src/main/java/core/selenium/WebElementAction.java

```diff
@@ -172,11 +171,47 @@ public String getSiteCurrentUrl() {
      * Clicks the checkbox elements.
      *
      * @param webElement checkbox.
-     * @param isPrivate set the status from private.
+     * @param stateOfCheckbox true to select, false to deselect.
      */
-    public void clickCheckBox(final WebElement webElement, final boolean isPrivate) {
-        if (!webElement.isSelected() && isPrivate) {
+    public void selectCheckBox(final WebElement webElement, final boolean stateOfCheckbox) {
+        if (!webElement.isSelected() && stateOfCheckbox) {
+            webElement.click();
+        }
+        if (webElement.isSelected() && !stateOfCheckbox) {
             webElement.click();
         }
     }
+
+    /**
+     * Gets the generic text from the header in a created opportunity page.
+     * @param xpath initial Xpath of the element.
+     * @param field name of the field to get the String.
+     * @param headerType type of the element field.
+     * @return the current text of the specific field.
+     */
+    public String getHeaderString(final String xpath, final String field, final String headerType) {
+        By byElement =  By.xpath(String.format(xpath, field, headerType));
+        if (isElementPresent(byElement, waitTime)) {
+            WebElement createdHeader = driver.findElement(byElement);
+            return createdHeader.getText();
+        } else {
+            return null;
+        }
+    }
+
+    /**
+     * Gets text from detail fields.
+     * @param xpath the path for the field.
```

Comment:
You should add a newLine to match the checkstyle convention.

Reply:
Done!